markdown/ml/ml1_perceptron/index.md

```diff
@@ -15,15 +15,52 @@ outcomes: false
 # assignments:
 #   - topic: blatt01
 #   - topic: blatt02
-# youtube:
-#   - id: XYZ (ID)
-#   - id: XYZ (ID)
+youtube:
+    - id: IJdiwITTC9Y
+    - id: oWcvFyLgqYc
+    - id: ZvWpI0Doocc
+    - id: 8Rdw2NBCCJk
+    - id: JD8Qsg8_kQI
 ---
 
 
 <div style="text-align:center;">
-Inhalt befindet sich im Aufbau<br>
-und wird rechtzeitig bereitgetellt.
+
+<h2>Kurze Übersicht</h2>
```

Comment:
@cyildiz warum schreibst du denn nicht einfach in markdown?

Reply:
Thanks for the reminder! Hatte das ganz vergessen :)